Form updates

taylorjohnson141 opened the pull request:
### Code Highlights:
 
### Where should the reviewer start?

form component
 
### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?
May need to edit form component

### Message/Questions for reviewer:

### Issues:
* Addresses #
* Closes #

### Screenshots (if appropriate):

### Tracking Consistency:
- [ ] added appropriate labels
- [ ] My code follows the code style of this project and has removed all unnecessary annotations
- [ ] I have added comments on my pull request, particularly in hard-to-understand areas
- [ ] looked at PR preview to check spelling, syntax, formatting, and completion

taylorjohnson141 committed:
Update question on click fuction

taylorjohnson141 committed:
Update skip button

taylorjohnson141 committed:
Conditionally render submit button

taylorjohnson141 committed:
Create Login page

taylorjohnson141 committed:
Minor updates

aemcdonald commented on .eslintcache:
```diff
@@ -0,0 +1 @@
+[{"/Users/taylor/turing/4module/capstone/eras-fe/src/index.tsx":"1","/Users/taylor/turing/4module/capstone/eras-fe/src/reportWebVitals.ts":"2","/Users/taylor/turing/4module/capstone/eras-fe/src/Components/App.tsx":"3","/Users/taylor/turing/4module/capstone/eras-fe/src/Components/FormContainer.tsx":"4","/Users/taylor/turing/4module/capstone/eras-fe/src/Components/Question.tsx":"5","/Users/taylor/t ...
```
I'm not sure we need to push up .eslintcache. From what I've read, it looks like it can be ignored but I will do more research.

aemcdonald commented on src/Components/App.tsx:
```diff
@@ -1,9 +1,20 @@
 export const App = () => {
   return(
-    <FormContainer />
+  <>
```
Awesome job getting us started with routes here.

aemcdonald commented on src/Components/FormContainer.tsx:
```diff
@@ -13,14 +13,17 @@ export const FormContainer = () => {
   const [answers, setAnswer] = useState<string[]>([]);
 
   const [currentQuestionIndex, changeQuestion] = useState(0)
+  console.log(answers)
```
I like the conditional logic & error handling here for only rendering the submit button on the last question

aemcdonald reviewed (changes requested):
Changes look great, but the Travis build is failing. Did you rebase before you pushed up? In the Travis job log, I am seeing that 1 test failed which should be passing on main right now.

taylorjohnson141 committed:
Add browser router to test

taylorjohnson141 commented:
The original App test did not have browser router. When I added the routes I did not add browser outer to the test which causes the test to fail. I fixed it!

krogowsk531 reviewed (approved):
Great job getting the initial calendar setup!

aemcdonald reviewed (approved):
Thanks for fixing the build!
